### PR TITLE
Cleanup UI for SIP gateway

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: css/filmstrip/_vertical_filmstrip.scss [hunk 1]
```diff
@@ -136,6 +136,13 @@
         display: flex;
         transition: opacity 1s;
     }
+
+    .hide-scrollbar#filmstripRemoteVideos {
+        margin-right: 7px; // Scrollbar size
+        &::-webkit-scrollbar {
```
Comment (hristoterezov) [inline css/filmstrip/_vertical_filmstrip.scss]: Out of curiosity - why hide the scrollbar exactly?

Comment (saghul) [reply css/filmstrip/_vertical_filmstrip.scss]: The user has no way to move it (they are on a SIP client) so it looks odd.

**Comment by saghul** [comment]
> Disable the whole notification feature, not only the rendering. There's no point of firing actions, storing into redux, etc if we won't render anything.

What do you have in mind? This is controlled with a config option, and we have no access to it in the reducer for example.

**Comment by hristoterezov** [comment]
Maybe we can check it in the actions and do no-op if needed. Although I'm not sure if this will be possible or what would be the impact after dispatch of an action that doesn't return anything. But an async action should be fine with this.

Another option would be to have enable/disable flag in the notifications part of the store which will be initialised by an action where we check everything needed to determine it.

**Comment by saghul** [comment]
I'm not sure it's worth it. We only do this for the SIP gateway, the other endpoints show notifiactions just fine, so there is noo performance concern here. Thus I went with the most straightforward and less invasive approach of hiding the visual notifications.

**Comment by hristoterezov** [comment]
Yep. My only concern would be for the performance of the sip gateway for a large meeting. It may have impact.

**Comment by saghul** [comment]
How is it different from a non SIP gateway meeting? If there is a problem it needs to be fixed in the regular case anyway.

**Comment by hristoterezov** [comment]
The difference is that we show the notifications in regular meeting and there's no way to optimize. For Sip gateway we don't, we can optimize and sip gateway will need less resources.

Just to clarify - this is not a problem or a thing I require from you. It was just a suggestion. If you don't want to do it or you don't think it will have impact this is fine.

**Comment by saghul** [comment]
> we can optimize and sip gateway will need less resources.

This needs numbers to back it up.

**Comment by hristoterezov** [comment]
What kind of numbers do you need? I really don't see what is the argument here about. Let's take this offline!

**Pull request merged**